02-create.md

```diff
@@ -32,7 +32,7 @@ $ ls
 ~~~
 
 But if we add the `-a` flag to show everything,
-we can see that Git has created a hidden directory called `.git`:
+we can see that Git has created a hidden directory within planets called `.git`:
```

Comment:
Fixed
Belinda


From: Daisie Huang [mailto:notifications@github.com]
Sent: Tuesday, 28 April 2015 6:15 AM
To: swcarpentry/git-novice
Cc: Belinda Weaver
Subject: Re: [git-novice] Adding change directory explanation (#100)


In 02-create.md<https://github.com/swcarpentry/git-novice/pull/100#discussion_r29186030>:

> @@ -32,7 +32,7 @@ $ ls

>  ~~~

>

>  But if we add the `-a` flag to show everything,

> -we can see that Git has created a hidden directory called `.git`:

> +we can see that Git has created a hidden directory within planets called `.git`:

This planets should also be in backticks.

—
Reply to this email directly or view it on GitHub<https://github.com/swcarpentry/git-novice/pull/100/files#r29186030>.